### DISCUSSION TITLE
Expected ',' or '}' after property value in JSON at position 274

### DISCUSSION THREAD

**Discussion by ericatoliveira** [frontendbr/forum#2381] [Perguntas e Respostas]
Pessoal, estou estudando on line e o tutor nos enviou todos os arquivos prontos, ou seja, deveriam rodar perfeitamente.
Porém quando mando executar uma função com um html simples, aparece este erro: 

**SyntaxError: Error parsing C:\Users\Erica\MODULO 3 - JAVASCRIPT\week-4\package.json: Expected ',' or '}' after property value in JSON at position 274**

Eu sou iniciante, estou aprendendo, não faço ideia do que seja isso. meu package.json nem tem essa posição 274 (acho).
Já procurei erros de sintaxe em tudo quanto foi lugar e só encontrei no package.json três críticas:

 {                                     // Property expected json(513)
    "main": "index.js",
    "type":"module",          // Trailing comma json(519)
  }
};                                    // End of file expected. json

O q posso fazer pra resolver isso?

**Comment by ninetails** [comment]
Oi @ericatoliveira,

Desculpe explicar talvez até demais mas quando disse iniciante eu não sei bem avaliar o quanto sabe ou não então vou tentar ser bem didática.

O JSON é uma forma para representarmos dados, tem uma particularidade que ele precisa ser meio que um objeto javascript semelhante quando a gente imprime na tela. Não suporta comentários e vírgulas ou ponto-e-vírgulas fora de contexto.

Usando teu arquivo de exemplo:

```json
{ // Property expected json(513)
"main": "index.js",
"type":"module", // Trailing comma json(519) <- esse aqui é uma vírgula "sobrando", que não deveria estar
}
}; // End of file expected. json <- aqui tem uma chaves sobrando, o que deixaria seu objeto json inválido.
  // Em cima abre uma vez e embaixo tá fechando duas vezes.
  // Também não precisa deste ponto-e-vírgula
```

Corrigindo o que falei e removendo os comentários, fica:

```json
{
  "main": "index.js",
  "type":"module"
}
```

E assim que vc vai salvar o arquivo `package.json`.

No Node.js, o arquivo `package.json` é bem peculiar, pois é quem define metadados de um pacote Node como nome, dependências etc.

Agora pra dar o erro numa posição além, creio que esse arquivo está sendo executado por algum programa ao qual desconheço, então se isso não a ajudar em algo, precisaria de mais informações 😢
